src/coreclr/jit/compiler.h

```diff
@@ -1871,10 +1871,10 @@ class Compiler
     DWORD expensiveDebugCheckLevel;
 #endif
 
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET || defined(FEATURE_HW_INTRINSICS)
```

Comment:
> but I never understand why this is `#if FEATURE_MULTIREG_RET ` and not `#ifdef FEATURE_MULTIREG_RET `

Because it's defined to 0/1.

> This seems to be set to 0 for windows/x64. It is slightly confusing to have `DivRem` that has multireg return on windows/x64 which doesn't support it?

`FEATURE_MULTIREG_RET` is about putting regular struct into multiple registers, while hw intrinsic is special and doesn't take part in regular ABI.